src/main/kotlin/com/kohttp/util/json.kt

```diff
@@ -0,0 +1,36 @@
+package com.kohttp.util
+
+fun json(init: Json.() -> Unit): String = Json().also(init).toString()
+
+class Json {
+    private val elements = mutableListOf<Pair<String, String>>()
+
+    fun json(init: Json.() -> Unit) = Json().also(init)
+
+    infix fun String.to(obj: List<*>) {
+        val transform: ((Any?) -> CharSequence)? = when {
+            obj.all { it is String } -> { { """"$it"""" } }
```

Comment:
how about checking whether each object is String or not? you never know what will be passed into it